.github/workflows/build.yml

```diff
@@ -52,6 +52,9 @@ jobs:
           java-version: '11'
           distribution: 'adopt'
 
+      - name: Add Homebrew to PATH
+        run: echo "/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin" >> $GITHUB_PATH
+
```

Comment:
Don't you need to add this to the release workflow too?

Reply:
:yikes: u right. done.